<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="TabWiget">
      <property name="styleSheet">
       <string notr="true">QTabWidget::tab-bar { alignment: center; }</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="login">
       <attribute name="title">
        <string>login</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>16777215</height>
              </size>
             </property>
             <widget class="QTextEdit" name="roomTx">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>180</y>
                <width>337</width>
                <height>30</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.SF NS Text'; font-size:13pt;&quot;&gt;123456789&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>24</x>
                <y>230</y>
                <width>30</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>user:</string>
              </property>
             </widget>
             <widget class="QPushButton" name="leaveRoomBtn">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>410</y>
                <width>337</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>离开房间</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="appIDText">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>280</y>
                <width>337</width>
                <height>30</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_23">
              <property name="geometry">
               <rect>
                <x>24</x>
                <y>290</y>
                <width>36</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>appID:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>24</x>
                <y>180</y>
                <width>30</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>room:</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="user">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>230</y>
                <width>337</width>
                <height>30</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
             <widget class="QPushButton" name="initBtn">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>330</y>
                <width>337</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>初始化</string>
              </property>
             </widget>
             <widget class="QPushButton" name="loginRoomBtn">
              <property name="geometry">
               <rect>
                <x>90</x>
                <y>370</y>
                <width>337</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>加入房间</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="User">
       <attribute name="title">
        <string>User</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>942</width>
             <height>673</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
